### PR TITLE
Responsive design

### PR DESCRIPTION
**Pull request by Andrew1995bono**
Media for different display sizes

### CODE REVIEW

File: src/app/components/homepage-components/header/header-info/header-info.component.scss [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+.logo {
+  @include _420 {
+    display: inline;
+    padding: 0;
+    width: 37%;
+  }
+  @include _420 {
+    width: 40%;
+  }
```
Comment (SStegniy) [inline src/app/components/homepage-components/header/header-info/header-info.component.scss:179]: _420 twice

File: src/app/components/homepage-components/navbar/input-field/input-field.component.scss [hunk 2]
```diff
@@ -1,15 +1,81 @@
...
+  @include _600 {
+    height: 55px;
+    justify-content: center;
+    padding: 0 45px;
+  }
+  @include _420 {
+    justify-content: center;
+    height: 40px;
+    padding: 0 15px;
+  }
```
Comment (SStegniy) [inline src/app/components/homepage-components/navbar/input-field/input-field.component.scss:13]: you dont need justify-content inside _420 media, it already works from _600

**Review by SStegniy — APPROVED** [review]

File: src/app/components/homepage-components/header/header-info/header-info.component.scss (hunk 1 above)
Comment (Andrew1995bono) [reply src/app/components/homepage-components/header/header-info/header-info.component.scss:179]: +fixed

File: src/app/components/homepage-components/navbar/input-field/input-field.component.scss (hunk 2 above)
Comment (Andrew1995bono) [reply src/app/components/homepage-components/navbar/input-field/input-field.component.scss:13]: fixed

**Pull request merged**